ansible/configs/satellite-vm/default_vars_ec2.yml

```diff
@@ -1,17 +1,14 @@
 ################################################################################
-################################################################################
-### Environment Structure
-################################################################################
+### Environment Settings for aws
 ################################################################################
 ## Environment Sizing
 
+cloud_provider: ec2                     # Which AgnosticD Cloud Provider to use                # User info for notifications
+HostedZoneId: Z3IHLWJZOU9SRT
+aws_region: ap-southeast-2
 
-default_key_name: ~/.ssh/{{key_name}}.pem
-tower_run: false
 
 # How many do you want for each instance type
-
-
 bastion_instance_type: "t2.medium"
 bastion_instance_image: RHEL75
```

Comment:
Why such an old version of RHEL?

Reply:
Satellite 6.4 supports only rhel 7 hence at time of writing config I used RHEL75, Now if I want to change then it requires more time to test bit by bit of satellite installation and configuration. So I will keep this version, may be later we will change.